backend/src/v1/complaint/complaint.controller.ts

```diff
@@ -93,4 +104,32 @@ export class ComplaintController {
   ): Promise<ZoneAtAGlanceStats> {
     return this.service.getZoneAtAGlanceStatistics(complaintType, zone);
   }
+
+  @Public()
```

Comment:
yes, we totally need the public guard here. Because the root controller has a `@UseGuards(JwtRoleGuard)` and the webeoc project done'st have a user we need to use the `@Public` decorator to override `@UseGuards(JwtRoleGuard)` and we use the `@UseGuards(ApiKeyGuard)` guard to make sure that the action is protected by an api key